### DISCUSSION TITLE
Add another new virtual hard disk to a virtual machine

### DISCUSSION THREAD

**Discussion by Mugen1982** [cirruslabs/tart#779] [General]
Hello all !
I would like to add other virtual hard disks to a Debian virtual machine (2CPU / 4GB RAM / 20GB HD0 / HD1 ?). Is it possible ?
Thank you in advance for your return.

**Comment by fkorotkov** [comment]
You can create disks manually and pass them via `--disk` argument of `tart run`.

**Reply by Mugen1982** [reply]
Thank you
